imports/api/apollo/schema.js

```diff
@@ -47,6 +51,21 @@ type Query {
 type Mutation {
   ${mutations}
 }
+
+type Email {
+  address: String
+  verified: Boolean
+}
+
+type UserProfile {
+  name: String
+}
+
+type User {
+  _id: String
+  emails: [Email]
+  profile: UserProfile
+}
```

Comment:
To save more info about somebody, like their access role, stick that info in the UserProfile